Improve JSON response

nick-gravgaard opened the pull request:
Change JSON to just return one property with the latest PPI data, as well as
property_type and fake coordinates

nick-gravgaard committed:
Improve JSON response

Change JSON to just return one property with the latest PPI data, as well as
property_type and fake coordinates

camillebaldock commented on app.py:
```diff
@@ -5,6 +5,14 @@
 app = Flask(__name__)
 
 
+PROPERTY_TYPES = {
```
Down the line, this could just be "the last part of the URL" instead of defining each one in a dictionary? :)

nick-gravgaard committed:
Create get_property_type() function

Get property type by partitioning URL rather than looking up in dict (for now).
